### PR TITLE
Update readmes to match common design (Feature Layers category) 3/x

### PR DESCRIPTION
**Pull request by JonLavi**

### CODE REVIEW

File: feature_layers/feature-layer-query/README.md [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
-## How to use the sample
+## Use case
+
+Query expressions can be used in ArcGIS to select a subset of features from a feature table. This is most useful in large or complicated data sets. A possible use case might be on a feature table marking the location of street furniture through a city. A user may wish to query by a TYPE column to return "benches". In this sample, we query a U.S. state by STATE_NAME from a feature table containing all U.S. states.
 
-Input the name of a U.S. state into search bar. When you hit search the application performs a query on the feature table and based on the result either highlights the state geometry or provides an error.
+## How to use the sample
 
-Note: The search is case sensitive.
+Input the name of a U.S. state into the text field. When you click "search", a query is performed and the matching features are highlighted or an error is returned.
```
Comment (Rachael-E) [inline feature_layers/feature-layer-query/README.md]:
```suggestion
Input the name of a U.S. state into the text field. When you click "Search", a query is performed and the matching features are highlighted or an error is returned.
```

File: feature_layers/feature-layer-rendering-mode-map/README.md [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
+
+In static rendering mode, features and graphics are rendered only when needed (for example, after an extent change) and offloads a significant portion of the graphical processing onto the CPU. As a result, less work is required by the GPU to draw the graphics, and the GPU can spend its resources on keeping the UI interactive. Use this mode for stationary graphics, complex geometries, and very large numbers of features or graphics. The number of features and graphics has little impact on frame render time, meaning it scales well, and pushes a constant GPU payload. However, rendering updates is CPU and system memory intensive, which can have an impact on device battery life.
+
+## How to use the sample
 
-To change `FeatureLayer.RenderingMode` using `LoadSettings`:
+Use the 'Animated Zoom' button to trigger the same zoom animation on both static and dynamically maps.
```
Comment (Rachael-E) [inline feature_layers/feature-layer-rendering-mode-map/README.md]: Our sample doesn't have an animated zoom button: it just triggers the zoom animation on launching.

File: feature_layers/feature-layer-rendering-mode-map/README.md [hunk 3]
```diff
@@ -1,31 +1,34 @@
...
-    * Can set preferred rendering mode for `Points`, `Polylines`, or `Polygons`.
-    * `Multipoint` preferred rendering mode is the same as point.
-3. Set map to `MapView`, `mapViewBottom.setMap(mapBottom)`.
-4. Create a `ServiceFeatureTable` from a point service, `new ServiceFeatureTable("http://sampleserver6.arcgisonline.com/arcgis/rest/services/Energy/Geology/FeatureServer/0");`.
-5. Create `FeatureLayer` from table, `new FeatureLayer(poinServiceFeatureTable)`.
-6. Add layer to map, `mapBottom.getOperationalLayers().add(pointFeatureLayer.copy())`
-    * Now the point layer will be rendered dynamically to map view.
+## How it works
+
+1. Create an `ArcGISMap` and call `getLoadSettings()` and then `setPreferred[Point/Polyline/Polygon]FeatureRenderingMode(...)`.
```
Comment (Rachael-E) [inline feature_layers/feature-layer-rendering-mode-map/README.md]: I prefer our original instructions to make the API read clearer: 

2. Set preferred rendering mode to map, `mapBottom.getLoadSettings().setPreferredPointFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC)`.	
    * Can set preferred rendering mode for `Points`, `Polylines`, or `Polygons`.

**Review by Rachael-E — APPROVED** [review]
only a few small things for this one, I'll approve so you can merge after making the changes!

File: feature_layers/feature-layer-rendering-mode-map/README.md (hunk 3 above)
Comment (JonLavi) [reply feature_layers/feature-layer-rendering-mode-map/README.md]: I've had a go at restructuring this section a little. Let me know what you think?

File: feature_layers/feature-layer-rendering-mode-map/README.md (hunk 2 above)
Comment (JonLavi) [reply feature_layers/feature-layer-rendering-mode-map/README.md]: Good catch, I've updated this section!

**Comment by JonLavi** [comment]
Hi @Rachael-E , thanks for the review. Can you have another look at the revised 'how it works' section please?

File: feature_layers/feature-layer-rendering-mode-map/README.md [hunk 4]
```diff
@@ -12,15 +12,17 @@ In static rendering mode, features and graphics are rendered only when needed (f
...
 
 ## How it works
 
-1. Create an `ArcGISMap` and call `getLoadSettings()` and then `setPreferred[Point/Polyline/Polygon]FeatureRenderingMode(...)`.
-2. The `RenderingMode` can be set to `STATIC`, `DYNAMIC` or `AUTOMATIC`.
-   * In Static rendering mode, the number of features and graphics has little impact on frame render time, meaning it scales well, however points don't stay screen-aligned and point/polyline/polygon objects are only redrawn once map view navigation is complete.
-   * In Dynamic rendering mode, large numbers of features or graphics can affect the responsiveness of maps or scenes to user interaction, however points remain screen-aligned and point/polyline/polygon objects are continually redrawn while the map view is navigating.
-3. When left to automatic rendering, points are drawn dynamically and polylines and polygons statically.
+1. Create an `ArcGISMap`
+2. Set the `RenderingMode` for a specific type of Graphic by calling `arcGISMap.getLoadSettings().setPreferredPointFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC)`.
```
Comment (Rachael-E) [inline feature_layers/feature-layer-rendering-mode-map/README.md]:
```suggestion
2. Set the `RenderingMode` for a specific type of `Graphic` e.g. for a `Point` by calling `arcGISMap.getLoadSettings().setPreferredPointFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC)`.
```

File: feature_layers/feature-layer-rendering-mode-map/README.md [hunk 5]
```diff
@@ -12,15 +12,17 @@ In static rendering mode, features and graphics are rendered only when needed (f
...
+    * You can set the preferred rendering mode for `Point`, Polylines, or Polygons by using the appropriate method.
```
Comment (Rachael-E) [inline feature_layers/feature-layer-rendering-mode-map/README.md]:
```suggestion
    * You can set the preferred rendering mode for `Point`, `Polyline`, or `Polygon` by using the appropriate method.    
```

**Review by Rachael-E — APPROVED** [review]

**Pull request merged**